package.json

```diff
@@ -29,9 +32,9 @@
     "wpcom-proxy-request": "^3.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "start": "eslint src && react-scripts start",
```

Comment:
Why lint before start? Seems more a pre-commit/deploy step.

Maybe add a `predeploy` script to `npm run test`